lib/util.js

```diff
@@ -634,8 +634,18 @@ const util = {
     }
 
     if (forceReset || util.shouldUpdateChromium()) {
-      args = [...args, ...chromiumArgs]
-      reset = true
+      // Separate shouldUpdate and shouldIgnore so
+      // that we still log the need to update.
+      if (!shouldIgnoreChromium) {
+        args = [...args, ...chromiumArgs]
+        reset = true
+      } else {
+        console.log(chalk.yellow.bold(`Not updating chromium since instructed to ignore via a flag.`))
+      }
+      // Only reset if forced to or we are performing a chromium update
+      if (forceReset) {
+        reset = true
```

Comment:
I fixed the logic here about setting `reset = true` if:
- `forceReset === true`
- `shouldUpdateChromium && !shouldIgnoreChromium`